packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -23,6 +23,9 @@ export type UiConfig = {
 type SetStyles = React.Dispatch<React.SetStateAction<EmbedStyles>>;
 type setNonStylesConfig = React.Dispatch<React.SetStateAction<EmbedNonStylesConfig>>;
 
+/**
+ * This is in memory persistence needed so that when user browses through the embed, the configurations from the instructions aren't lost.
```

Comment:
```suggestion
 * This is in-memory persistence needed so that when user browses through the embed, the configurations from the instructions aren't lost.
```